Jucer2Reprojucer/main.cpp

```diff
@@ -1366,12 +1370,12 @@ int main(int argc, char* argv[])
       wLn("  PATH \"",
           useGlobalPath ? (isJuceModule ? "${JUCE_MODULES_GLOBAL_PATH}"
                                         : "${USER_MODULES_GLOBAL_PATH}")
-                        : relativeModulePath.replace("\\", "/"),
+                        : modulePath.replace("\\", "/"), //
```

Comment:
Is the comment superfluous? Otherwise looks good :)

Reply:
Very good catch! I added that comment to track down the origin of the `relativeModulePath` variable using `gitk`'s "Origin of this line" feature. I simply forgot to remove it before making the commit :sweat_smile:

Reply:
Addressed in 0547d00b8762d96ec83194ccc5af1e3305279994.